manifests/module.pp

```diff
@@ -71,14 +75,20 @@
     default => $module_prefix,
   }
 
+  $real_install_options = $install_options ? {
+    ''      => $php::install_options,
+    default => [],
```

Comment:
Then place [] as default value in the parameter list, otherwise the user's value, if given, is ignored